### ISSUE TITLE
Create S3 bucket for NAMC GIS data

### ISSUE THREAD

**Issue by philipbaileynar** [namc-utah/namc-bug-db#240]
Please create an S3 bucket for NAMC to store and share GIS data.

- This is intended for internal NAMC use only (i.e. no public sharing)
- I envisage 3-6 NAMC staff using this repository of GIS data.
- The data will not change be changing frequently. It is intended as a long term storage of fairly static data.
- I envisage (at least for the short term) NAMC staff having a batch file that they click in Windows explorer to perform sync to their local machines. We can discuss the correct flags for contribute versus delete etc.
- We need documentation so that NAMC staff know how to install AWS CLI and also how NAMC system admin can grant/revoke access to the S3 bucket. In a dream world, this authorization could be performed through the React Web App?

In plain English.... NAMC has a dozen or more large GIS datasets (raster and ShapeFile) that analysts refer to both from desktop GIS and R etc. The goal is to move this data off their Windows File Server and onto S3 where analysts can perform periodic sync to retrieve the latest version so they can work offline and refer to these data.

FYI @Feldspar5

**Comment by MattReimer** [comment]
@David-Fowler I've got a bucket with two roles to read and read/write objects and I've got corresponding Cognito groups with a user in each.

For the life of me though I can't figure out how to configure rclone remotes using cognito. All the S3 settings seem to ask for IAM credentials and [the s3 docs](https://rclone.org/s3/) don't mention Cognito or OAuth. 

Before I go off on a goose chase do you have any thoughts about where I could look for some help here?
